classes/03class/exercises/c03-docker07/README.md

```diff
@@ -0,0 +1,5 @@
+# ECS (c03-docker07)
+TBD
```

Comment:
WIP

Reply:
waiting for the lb/asg class to check how far I can go with this exercise.